lib/crypto.js

```diff
@@ -633,6 +633,37 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
+function randomFillSync(buf, offset = 0, size) {
+  if (!Buffer.isBuffer(buf)) {
+    throw new TypeError('"buf" argument must be a Buffer');
+  }
+
+  if (!size) size = buf.length - offset;
```

Comment:
I know `size === 0` is kind of an edge case but I’d actually expect that to be a no-op. So maybe a strict comparison with `undefined` is better than `!size`?